portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,81 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.flex-container {
+	display: flex;
+}
```

Comment:
nit: Remove a few spaces before "display". keep the tabs consistent for properties across this file. Makes the file more readable.